gatsby/src/components/Layout/index.js

```diff
@@ -1,10 +1,26 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { Grid, Box, ResponsiveContext } from "grommet";
+import { Grid as GrommetGrid, Box, ResponsiveContext } from "grommet";
 import { AxisTheme } from "@centrifuge/axis-theme/";
+import styled from "styled-components";
 
 import Nav from "../Nav";
 
+const Grid = styled(GrommetGrid).attrs({
+  margin: { horizontal: "auto" },
+  fill: true
+})`
+  max-width: calc(1152px - 32px);
```

Comment:
This should be more for larger screens and the default should have sidebars span 2 columns I think

Reply:
we can iron this out later.